.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: python
 python:
   - '2.7'
+services:
+  - elasticsearch
+before_script:
+  - sleep 10
```

Comment:
So this is to wait for elasticsearch to start? Kind of wish heroku handled this automatically if it is.